src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+body,
```

Comment:
These styles are not needed. We only need to reset margin for `body`.
Also, according to CSS coding guideline, we should not use tags in CSS selectors, except `html` and `body`.